sru-templates/manual/ec2-sru

```diff
@@ -15,6 +16,32 @@ else
     PROPOSED_SCRIPT=setup_dev_proposed.sh
 fi
 
+while [ $# -ne 0 ]; do
+	case "$1" in
+		# any flag pass through.
+		--preserve-instance) PRESERVE_INSTANCE=true;;
+		--) shift; break;;
+	esac
+	shift;
+done
+
+get_ec2_instance_id() {
+    dns_name=$1
+    dns_name="${dns_name#"ubuntu@"}"
+    query='Reservations[].Instances[].InstanceId'
+    aws ec2 describe-instances --filter "Name=dns-name,Values=${dns_name}" --query $query | jq .[]
+}
+
+terminate_instances() {
+    ec2_inst=$1
+    if [ "$PRESERVE_INSTANCE" = false ]; then
+        echo "deleting instance"
+        ec2_inst_id=`get_ec2_instance_id $ec2_inst | tr -d '"'`
```

Comment:
can drop the tr -d if you use jq -r above.   Also, best to use curly braces around params for better visibily

```suggestion
        ec2_inst_id=`get_ec2_instance_id ${ec2_inst}`
```